keyboards/levinson/keymaps/iarumas/neo2-common.h

```diff
@@ -0,0 +1,14 @@
+#ifndef NEO2_COMMON_H
+#define NEO2_COMMON_H
+
+
+/*
+This file holds some commen NQN definitions
+*/
+
+
+#define _______ KC_TRNS
```

Comment:
Lines 10 and 11 here are already defined in the QMK core automatically, so these definitions are redundant here.

https://github.com/qmk/qmk_firmware/blob/7c63c0a95075f6fb4dd94d020ceb5816e6bcca2c/quantum/quantum_keycodes.h#L29-L31

That being the case, you could refactor your headers to remove this entire file and be no worse off.

Same may be possible for `users/iarumas/nqn-common.h` for the same reason, but as I have no experience with the userspace feature, I'm not 100% on that.